frontend_lib/src/css/Variable.styl

```
@@ -27,13 +27,24 @@ zIndexOverHeader = 50
 headerHeight = 61px // INFO - CH - 2019-04-19 - the total header height. Used to calculate height of layout like height: "calc(100% - %s)" % headerHeight
 sidebarWidth = 300px
 
-fontSizeTitle = 25px
-fontSizeMedium = 20px
-fontSizeSmall = 14px
+fontStandard = 16px
+mainTitle = 25px
+mediumTitle = 20px
+smallTitle = 14px
```

Leave "fontSize" to clearity. 

Just a suggestion: 

```suggestion
fontSizeStandard = 16px
fontSizeTitle = 25px
fontSizeSubtitle = 20px
fontSizeMetadata = 14px
```

---

Should we stay with the same values? I was thinking 14px as standard, 10 or 12px for metadatas and 20 for titles (we need a subtitle?)